### PR TITLE
style(components): [tour] Improve style and docs

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/theme-chalk/src/tour.scss [hunk 1]
```diff
@@ -63,11 +62,11 @@
...
     @include e(closebtn) {
       position: absolute;
-      top: 6px;
+      top: 0;
       right: 0;
       padding: 0;
-      width: 44px;
-      height: 44px;
+      width: 40px;
+      height: 40px;
```
Comment (kooriookami) [inline packages/theme-chalk/src/tour.scss:69]: image

File: packages/theme-chalk/src/tour.scss [hunk 2]
```diff
@@ -156,14 +159,9 @@
 
     .#{$namespace}-button--default {
       color: getCssVar('color', 'primary');
-      border-color: getCssVar('color', 'primary');
       background: #fff;
     }
 
-    .#{$namespace}-button--primary {
-      border-color: #fff;
```
Comment (Fuphoenixes) [inline packages/theme-chalk/src/tour.scss:164]: 按钮的border去掉后，样式变了。
image

File: packages/theme-chalk/src/tour.scss [hunk 3]
```diff
@@ -156,14 +159,9 @@
 
     .#{$namespace}-button--default {
       color: getCssVar('color', 'primary');
-      border-color: getCssVar('color', 'primary');
```
Comment (Fuphoenixes) [inline packages/theme-chalk/src/tour.scss:159]: image

File: packages/theme-chalk/src/tour.scss (hunk 2 above)
Comment (kooriookami) [reply packages/theme-chalk/src/tour.scss:164]: 这个我没注意改错了，不过按钮的border手动设置有点怪哈。

Comment (Fuphoenixes) [reply packages/theme-chalk/src/tour.scss]: 嗯，但是不加border有点丑

**Review by Fuphoenixes — APPROVED** [review]

File: packages/theme-chalk/src/tour.scss (hunk 2 above)
Comment (kooriookami) [reply packages/theme-chalk/src/tour.scss]: 当初这个primary的属性不要加比较好

**Comment by kooriookami** [comment]
If it's OK, please merge it.

**Pull request merged**